README.md

```diff
@@ -1,2 +1,136 @@
+
+1. Adding octopus action will add ability to set up continuous delivery to octopus. This action can be invoked by action name and release version.
+
+```yaml
+
+    - name: Lazy Action Octopus
+      uses: variant-inc/actions-octopus@v1
+      with:
+        default_branch: ${{ env.MASTER_BRANCH }}
+        deploy_scripts_path: deploy
+        project_name: ${{ env.PROJECT_NAME }}
+        version: ${{ steps.lazy-setup.outputs.image-version }}
+        space_name: ${{ env.OCTOPUS_SPACE_NAME }}
+
+```
+
+Refer [octopus action](https://github.com/variant-inc/lazy-action-octopus/blob/master/README.md) for documentation.
+
+## Using Python Action
+
+You can set up continuous integration for your project using a lazy workflow action.
+After you set up CI, you can customize the workflow to meet your needs. By passing the right input parameters with the actions python.
+
+### Adding actions-python to workflow
+
+Sample snippet to add lazy action to your workflow code.
+See [action.yml](action.yml) for the full documentation for this action's inputs and outputs.
+
+```yaml
+jobs:
+  build_test_scan:
+    runs-on: eks
+    name: CI Pipeline
+    steps:
+    - uses: actions/checkout@v2
+      with:
+        fetch-depth: 0
+
+    - name: Setup
+      uses: variant-inc/actions-setup@v1
+
+    - name: Lazy action steps
+      id: lazy-action
+      uses: variant-inc/actions-python@v1
+      env:
+        AWS_DEFAULT_REGION: us-east-1
+        GITHUB_USER: variant-inc
+        AWS_REGION: us-east-1
+      with:
+        dockerfile_dir_path: '.'
+        ecr_repository: naveen-demo-app/demo-repo
+        github_token: ${{ secrets.GITHUB_TOKEN }}
+
+    - name: Lazy Action Octopus
+      uses: variant-inc/actions-octopus@v1
+      with:
+        default_branch: ${{ env.MASTER_BRANCH }}
+        deploy_scripts_path: deploy
+        project_name: ${{ env.PROJECT_NAME }}
+        version: ${{ steps.lazy-setup.outputs.image-version }}
+        space_name: ${{ env.OCTOPUS_SPACE_NAME }}
+
+```
+
+### Input Parameters
+
+| Parameter                     | Default         | Description                                                                                                                  | Required |
+| ----------------------------- | --------------- | ---------------------------------------------------------------------------------------------------------------------------- | -------- |
+| `dockerfile_dir_path`         | `.`             | Directory path to the dockerfile                                                                                             | true     |
+| `ecr_repository`              |                 | ECR Repository name                                                                                                          | true     |
+| `github_token`                |                 | Github Token                                                                                                                 | true     |
```

Comment:
You don't have github_token in your action.yml input parameters.

Reply:
will add